README.md

```diff
@@ -24,6 +24,39 @@ For more information on prow, see the upstream [documentation](https://github.co
 
 A Prometheus server runs in the CI cluster and is configured to create [alerts](https://prometheus-kube-system.svc.ci.openshift.org/alerts) on top of prow metrics. By clicking on the `expr` field of every alert, you can view the query that is setup for alerting. For more information on alerts, see [the Prometheus docs](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/).
 
+Possible reactions to some of these alerts:
+
+* Slow Jenkins operator sync
+* Slow Jenkins pipeline operator sync
+
+These should not be a problem in general but if any of them persists for more than a couple of hours, [`max_goroutines`](https://github.com/openshift/release/blob/ff18182aa0eb849b89e7abd1bc7765ad6d27142f/cluster/ci/config/prow/config.yaml#L7) can be incremented to allow more parallelism in the operators (note that the same option dictates both operators).
```

Comment:
Could also be Jenkins master being slow to respond, overloaded, etc?

Reply:
Yeah, I need to craft a query that uses `jenkins_requests_latency` so we can figure out if Jenkins is the culprit.

Reply:
Opened https://github.com/openshift/release/pull/485

Note though that it's unlike we are going to have slow syncs ever since https://github.com/kubernetes/test-infra/pull/5774